lib/core/collections.toit

```diff
@@ -1649,6 +1660,17 @@ abstract class ByteArrayBase_ implements ByteArray:
   write-to-byte-array target/ByteArray --at/int from/int to/int -> none:
     target.replace at this from to
 
+  reverse --in-place/True -> none:
```

Comment:
Add a few simple tests for these two new functions?

Reply:
done.